package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "factorio_blueprint_editor",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "description": "Online Editor for Elxeno's Factorio Blueprint Editor",
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "./node_modules/.bin/eslint js/**/*.js"
+    "lint": "eslint js/**/*.js"
```

Comment:
we should keep using the project's version of eslint instead of relying on a globally specified version

Reply:
npm handles this already, the `./node_modules/.bin` is redundant.

It basically puts `./node_modules/.bin` at the front of your `PATH` so we are using project's dependencies before anything installed system wide.

The official docs are very terse on this: https://docs.npmjs.com/misc/scripts#path

Reply:
Oh. I did not know that. Nvm then